resources/app.py

```diff
@@ -385,6 +491,322 @@ def delete_bed_reservation(patient_id: str, db=Depends(get_db)):
     return {"message": "Bed reservation deleted successfully."}
 
 
+# ======================== Employee Management ========================
+@app.get("/get/employee/all", tags=["Employee"])
```

Comment:
I know i said you that we don't need creating empleyee endpoint, but as i digged into this, we need. Very similiar as `create_patient`. It will be only for admin.
Sorry for my misinformation

Reply:
Rest of the endpoints are untestable if there is no employee in db

Reply:
> I know i said you that we don't need creating empleyee endpoint, but as i digged into this, we need. Very similiar as `create_patient`. It will be only for admin. Sorry for my misinformation.

does we dont want to create employes and other 'privilidged' users only from keycloack (UI + admin account)?

Reply:
> I think we should also add role when creating patient/doctor, it will be useful for example in `create/surgical_plans` to check if providen uuid is a real doctor_id

We (if i understand correctly how KC works) also need to define roles in JSON file for keycloak realms. Espcially when we plan to use them in code

Reply:
i thought about our ui and only admin role can create
remember we have employee related data in our db

Reply:
but we have doctor already

Reply:
> i thought about our ui and only admin role can create remember we have employee related data in our db

I dont se benefits from this, but if you i have zero problemas amigo.